File: codewars/python/Two Sum.py
def two_sum(numbers, target):
    for i in numbers:
        for j in numbers:
            if i+j==target:
                if i!=j:
                    return [numbers.index(i), numbers.index(j)]
                else:
                    return [k for k,x in enumerate(numbers) if x==i]

File: leetcode/java/Two Sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] ans = new int[2];
        Map<Integer, Integer> indMap = new HashMap<>();
        for (int i = 0; i<nums.length; i++){
            if (indMap.containsKey(target - nums[i])){
                ans[0] = i;
                ans[1] = indMap.get(target - nums[i]);
            }
            indMap.put(nums[i], i);
        }
        return ans;
    }
}